Author: Jeremy Sowden <jeremy@azazel.net>
Subject: Clean some extra build artefacts
Last-Update: 2020-08-29
Forwarded: https://lore.kernel.org/netfilter-devel/20200829204127.2709641-1-jeremy@azazel.net/
Description: Add extensions/ACCOUNT and extensions/pknock to `SUBDIRS`
 to ensure their Makefiles are cleaned by `make distclean`.
 .
 Add .manpages.lst to Makefile.mans's `clean` target.

diff --git a/Makefile.mans.in b/Makefile.mans.in
index 18e82b39882d..63424f7d558a 100644
--- a/Makefile.mans.in
+++ b/Makefile.mans.in
@@ -40,4 +40,4 @@ targets.man: .manpages.lst ${wcman_targets}
 	$(call man_run,${wlist_targets})
 
 clean:
-	rm -f xtables-addons.8 matches.man targets.man
+	rm -f xtables-addons.8 matches.man targets.man .manpages.lst
diff --git a/Makefile.am b/Makefile.am
index 75c7c04ef95d..e62be080d829 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,7 +1,7 @@
 # -*- Makefile -*-
 
 ACLOCAL_AMFLAGS  = -I m4
-SUBDIRS          = extensions geoip
+SUBDIRS          = extensions extensions/ACCOUNT extensions/pknock geoip
 
 man_MANS := xtables-addons.8
 
-- 
2.28.0
